api/base/filters.py

```diff
@@ -24,14 +24,19 @@
 def sort_multiple(fields):
     fields = list(fields)
```

Comment:
Kwame has already worked on this https://github.com/CenterForOpenScience/osf.io/pull/5258, and it's marked for .68.  This has already been reviewed so I would use this.  It will make your institution registrations sort properly.